lib/london_bike_hire_cli/feed_parser.rb

```diff
@@ -3,7 +3,7 @@
 
 module LondonBikeHireCli
   class FeedParser
-    TFL_FEED_URL = 'http://www.tfl.gov.uk/tfl/syndication/feeds/cycle-hire/livecyclehireupdates.xml'.freeze
+    TFL_FEED_URL = 'https://tfl.gov.uk/tfl/syndication/feeds/cycle-hire/livecyclehireupdates.xml'.freeze
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.